README.md

```diff
@@ -5,16 +5,16 @@ Dispatch jobs inside transactions. Cancel job on transaction rollback. Add to qu
 
 1) Run ```composer require therezor/laravel-transactional-jobs``` in your laravel project root folder
 
-2) Add `public $afterTransactions = true;` property to jobs that runs in the middle of transactions
```

Comment:
That was made intensionally to switch this property on fly. It is a good idea to add Interface, but better keep both for compitability reasons. Also naming of interface should be more understandable (like "RunAfterTransactionCommit")